assets/mods/ccloader-version-display/mod.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+	function createCCLoaderVersionText() {
+		const LOADER_NAME = "CCLoader";
+	 
+		let text = new cc.sc.Text(LOADER_NAME + " V" + versions.ccloader, {
+			font: cc.sc.fontsystem.tiny
+		});
+		text[entries.setPosition](2, 10);
+		text[entries.setAlignment](6, 3);
+		
+		text[entries.GuiConfig][entries.transition] = {
+            DEFAULT: {
+                state: {},
+                time: 0.2,
+                p: KEY_SPLINES.EASE
+            },
+            HIDDEN: {
+                state: {
+                    alpha: 0
+                },
+                time: 0.2,
+                p: KEY_SPLINES.LINEAR
+            }
+        };
+		return text;
+	}
+	const onModsLoaded = function() {
+		let titleBG = cc.ig.GUI.menues.filter((e) => e[entries.GUI] instanceof cc.sc.TitleScreenBG).pop();
+		if(!titleBG) {
+			throw new Error('Could not find titleBG');
```

Comment:
Does this actually happen? If so, please include an easier to understand error message for the end user.